src/meta-srv/src/table_routes.rs

```diff
@@ -185,14 +154,21 @@ pub(crate) mod tests {
         // 1 => 1, 2
         // 2 => 3
         // 3 => 4
+        let region_distribution =
+            RegionDistribution::from([(1, vec![1, 2]), (2, vec![3]), (3, vec![4])]);
         table_metadata_manager
             .table_region_manager()
-            .put_old(
-                &TableName::new(DEFAULT_CATALOG_NAME, DEFAULT_SCHEMA_NAME, table),
-                RegionDistribution::from([(1, vec![1, 2]), (2, vec![3]), (3, vec![4])]),
-            )
+            .create(1, &region_distribution)
             .await
             .unwrap();
+
+        for (datanode_id, regions) in region_distribution {
```

Comment:
Considers creating in parallel?

Reply:
it's in tests, backed by memory kvbackend, should be fast